%!TEX TS-program = lualatex
%!TEX encoding = UTF-8 Unicode

\documentclass[t]{beamer}

%%%% HANDOUTS For online Uncomment the following four lines for handout
%\documentclass[t,handout]{beamer}  %Use this for handouts.
%\usepackage{handoutWithNotes}
%\pgfpagesuselayout{3 on 1 with notes}[letterpaper,border shrink=5mm]

%\includeonlylecture{student}

%%% Including only some slides for students.
%%% Uncomment the following line. For the slides,
%%% use the labels shown below the command.

%% For students, use \lecture{student}{student}
%% For mine, use \lecture{instructor}{instructor}

% FONTS
\usepackage{fontspec}
\def\mainfont{Linux Biolinum O}
\setmainfont[Ligatures={Common,TeX}, Contextuals={NoAlternate}, BoldFont={* Bold}, ItalicFont={* Italic}, Numbers={OldStyle}]{\mainfont}
\setsansfont[Ligatures={Common,TeX}, Scale=MatchLowercase, Numbers=OldStyle, BoldFont={* Bold}, ItalicFont={* Italic},]{Linux Biolinum O} 
\usepackage{microtype}

\usepackage{graphicx}
	\graphicspath{{/Users/goby/pictures/teach/163/lecture/}
	{/Users/goby/pictures/teach/common/}} % set of paths to search for images

\usepackage{multicol}
\usepackage{booktabs}
\usepackage{array}
\newcolumntype{L}[1]{>{\raggedright\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{R}[1]{>{\raggedleft\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
%\usepackage{textcomp}
%\usepackage{mhchem}
\usepackage{enumitem}
%\usepackage[export]{adjustbox}


\usepackage{tikz}
	\tikzstyle{every picture}+=[remember picture,overlay]
	\usetikzlibrary{arrows}
\usetikzlibrary{positioning}

\mode<presentation>
{
  \usetheme{Lecture}
  \setbeamercovered{invisible}
}


\begin{document}

\lecture{student}{student}

\begin{frame}{Our goal for this lecture is to learn about}
	
	\hangpara \highlight{molecular clocks}, and
	
	\hangpara genome evolution,
	
	\hangpara the evolutionary role of \highlight{homeotic genes} and other developmental genes.

\end{frame}
%
\begin{frame}[b]{Mutations occur at a \emph{roughly} constant rate, called a \highlight{molecular clock.}}

\begin{tikzpicture}
	\tikzset{every node/.style={font=\footnotesize}};
	\node (ca) [align=right] at (1,5){\footnotesize Common ancestor:\\ \textsc{caatttatcg}};
	
	\draw [thick] (ca.east) -- ++(15:8cm) 
		node [midway, above left, align=right](25up)
			{25 \textsc{my} later \\\textsc{caattt\textbf{\textcolor{orange6}{g}}tcg}} 
		node [right, above left, align=right](50up)
			{50 \textsc{my} later\\ \textsc{caat\textbf{\textcolor{orange6}{a}}t\textbf{\textcolor{orange6}{g}}tcg}};
	\draw [thick] (ca.east) -- ++(-15:8cm) 
		node [midway, below left, align=right](25down)
			{\textsc{caa\textbf{\textcolor{orange6}{c}}ttatct}\\25 \textsc{my} later} 
		node [right, below left, align=right](50down)
			{\textsc{caa\textbf{\textcolor{orange6}{c}}ttatc\textbf{\textcolor{orange6}{t}}}\\50 \textsc{my} later};
	
%	\draw 
	\node (arrowbase1) at (3,5){};
	\node (arrowbase2) [below=of 25up.east,xshift=0.5cm,yshift=0.639cm]{};
	\node (arrowbase3) [above=of 25down.east,xshift=0.5cm,yshift=-0.639cm]{};
	
	\draw [thick] (25up.east) -- ++(-90:0.45cm);
	\draw [thick] (25down.east) -- ++(90:0.45cm);

	\draw [thick] (50up.east) -- ++(-90:0.45cm);
	\draw [thick] (50down.east) -- ++(90:0.45cm);

	\draw [->] (arrowbase1.center) -- ++(15: 3.3cm) node (arrow1){};
	\draw [->] (arrowbase1.center) -- ++(-15: 3.3cm) node (arrow2){};
	
	\draw [->] (arrowbase2.center) -- ++(15: 3.4cm);
	\draw [->] (arrowbase3.center) -- ++(-15: 3.4cm);
	
\end{tikzpicture}

\end{frame}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{molecular_clock_calibration} }
\begin{frame}[b]{Molecular clocks have to be calibrated to the fossil record or known geological events.}

\hfill \tiny Bromham and Penny 2003. Nature Reviews Genetics 4: 216.
\end{frame}
}
%
\begin{frame}[t]{If two species differ by 30 mutations, when did they last share a common ancestor?}

	\includegraphics[width=\textwidth]{molecular_clock_mammals}
	
	\onslide<2>{\vspace*{0.5\baselineskip}How many mutations if they last shared a common ancestor about 60 \textsc{mya}?}

	\vskip0pt plus 1filll
	
	\hfill \tiny \textcopyright Pearson Education, Inc.
\end{frame}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{molecular_clock_genes} }
\begin{frame}[b]{Different genes have different clock rates.}

\end{frame}
}
%
\begin{frame}{What genetic changes explain macroevolutionary change?}

	\hangpara If the fossil record documents macroevolutionary change, 
	then the \highlight{genomes} of the descendants should document those changes.
	 
\end{frame}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{gene_duplication} }
\begin{frame}[b]
\end{frame}
}
%
\begin{frame}{The globin family evolved by gene duplication.}

	\centering\includegraphics[height=0.82\textheight]{gene_duplication_globins}
	 
	 \vfilll
	 
	 \hfill \tiny \textcopyright Pearson Education, Inc.
\end{frame}
%

%% Orthologous and paralogous was too complicated. No need for it.

%\begin{frame}{Two types of \highlight{homologous genes} are found in the genome.}
%
%	\includegraphics[width=\textwidth]{gene_duplication_homologs}
%	 
%	 \vfilll
%	 
%	 \hfill \tiny \textcopyright Pearson Education, Inc.
%\end{frame}
%%
%\begin{frame}
%
%	\begin{overprint}
%		\alt<handout>{}{\onslide<1>\includegraphics[height=0.78\textheight]{para_ortho_1}
%		\onslide<2>\includegraphics[height=0.78\textheight]{para_ortho_2}
%		\onslide<3>\includegraphics[height=0.78\textheight]{para_ortho_3}
%		\onslide<4>\includegraphics[height=0.78\textheight]{para_ortho_4}
%		\onslide<5>\includegraphics[height=0.78\textheight]{para_ortho_5}
%		\onslide<6>\includegraphics[height=0.78\textheight]{para_ortho_6}}
%		\onslide<7>\includegraphics[height=0.78\textheight]{para_ortho_7}
%	\end{overprint}
%	
%\end{frame}
%%
%\begin{frame}[t]{Identify some paralogs and homologs.}
%	\includegraphics[width=\textwidth]{gene_duplication_phylogeny}
%	
%	\vfilll
%	
%	\hfill \tiny Ballerini and Kramer 2011. Frontiers in Plant Science 2(81): 1.
%\end{frame}
%
\begin{frame}[t]{Duplicated genes can become \highlight{pseudogenes} or evolve new functions.}
	
	\includegraphics[width=\textwidth]{gene_duplication_pseudogenes_opsins}\\
	
	\vfilll
	 
	\hfill \tiny Top: \textcopyright Pearson Education, Inc. Bottom: \textcopyright Oxford University Press.
\end{frame}
%
\begin{frame}[t]{Olfactory receptors in vertebrates have hundreds of homologous genes.}
	
	\includegraphics[width=\textwidth]{gene_duplication_olfactory_receptors}\\
	
	\vfilll
	 
	\hfill \tiny Niimura et al. 2014. Genome Research 24: 1485.
\end{frame}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{homeotic_genes_hox} }
\begin{frame}[b]

	\hfill \tiny \href{http://learn.genetics.utah.edu/basics/hoxgenes}{learn.genetics.utah.edu}
\end{frame}
}
%
\begin{frame}[t]{The role of homeotic genes was discovered through their mutations.}
	\vspace*{-0.5\baselineskip}
	\begin{center}
		\includegraphics[width=0.85\textwidth]{homeotic_genes_antp}
	\end{center}
	
	A mutation to the \textit{Antp} \textit{Hox} gene in Drosophila causes legs to grow where the antennae should be.
	\vfilll

	\hfill \tiny Fig. 18.20 \textcopyright Pearson Education, Inc.
\end{frame}
%
\begin{frame}[t]{Homeotic proteins regulate other genes.}
	\vspace*{-0.5\baselineskip}
	\begin{center}
		\includegraphics[width=0.75\textwidth]{homeotic_genes_regulation}
	\end{center}
	
	If the binding sequence changes, the gene may be ``turned on'' in the wrong place or ``turned off.''
	\vfilll

	\hfill \tiny \href{http://learn.genetics.utah.edu/basics/hoxgenes}{learn.genetics.utah.edu}
\end{frame}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{homeotic_genes_hox_fly} }
\begin{frame}[b]{}

	\tiny \href{http://learn.genetics.utah.edu/basics/hoxgenes}{learn.genetics.utah.edu}
\end{frame}
}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{homeotic_genes_inverts} }
\begin{frame}[b]{\textit{Ubx} is a \textit{Hox} gene that controls leg position in invertebrates.}

	\hfill \tiny Fig. 25.25 \textcopyright Pearson Education, Inc.
\end{frame}
}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{homeotic_genes_arthropod_diversity} }
\begin{frame}[b]{}

	\tiny \href{http://learn.genetics.utah.edu/basics/hoxgenes}{learn.genetics.utah.edu}
\end{frame}
}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{homeotic_genes_vertebrates} }
\begin{frame}[b]{\textit{Hox genes} also control the vertebrate body plan.}

	\tiny \href{http://learn.genetics.utah.edu/basics/hoxgenes}{learn.genetics.utah.edu}
\end{frame}
}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{homeotic_genes_vertebrate_duplication} }
\begin{frame}[b]

\end{frame}
}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{eyes_structure} }
\begin{frame}{Eyes evolved independently multiple times.}

\begin{minipage}{0.27\textwidth}
	
	\flushright
	\vspace{2em}
	\hangpara Insect
	
	\vspace{5em}
	
	\hangpara Giant clam	
	\vspace{5em}
	
	\hangpara Human
	
\end{minipage}
\end{frame}
}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{eyes_pax6} }
\begin{frame}[t]{The \highlight{\textit{Pax6}} developmental gene controls eye development.}

	\vfilll
	
	\tiny Carroll et al. 2005. \textit{From DNA to Diversity}. Blackwell.
\end{frame}
}
%
{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{homeotic_genes_paedomorphosis} }
\begin{frame}[b]

	\tiny Fig. 25.23 \textcopyright Pearson Education, Inc.
\end{frame}
}
%
\end{document}
