%!TEX TS-program = lualatex
%!TEX encoding = UTF-8 Unicode

\documentclass[12pt]{exam}
\usepackage{graphicx}
	\graphicspath{{/Users/goby/Pictures/teach/163/lab/}} % set of paths to search for images

\usepackage{geometry}
\geometry{letterpaper, bottom=1in}                   

\usepackage{afterpage}
\usepackage{pdflscape}

\newlength{\myindent}
\setlength{\myindent}{\parindent}
\newcommand{\ind}{\hspace*{\myindent}}


%\geometry{landscape}                % Activate for for rotated page geometry
\usepackage[parfill]{parskip}    % Activate to begin paragraphs with an empty line rather than an indent
%\usepackage{amssymb, amsmath}
%\usepackage{mathtools}
%	\everymath{\displaystyle}

\usepackage{fontspec}
\setmainfont[Ligatures={TeX}, BoldFont={* Bold}, ItalicFont={* Italic}, BoldItalicFont={* BoldItalic}, Numbers={OldStyle,Proportional}]{Linux Libertine O}
\setsansfont[Scale=MatchLowercase,Ligatures=TeX, Numbers=OldStyle]{Linux Biolinum O}
\setmonofont[Scale=MatchLowercase]{Inconsolatazi4}
\usepackage{microtype}

\usepackage{unicode-math}
\setmathfont[Scale=MatchLowercase]{Asana Math}
%\setmathfont[Scale=MatchLowercase]{XITS Math}

% To define fonts for particular uses within a document. For example, 
% This sets the Libertine font to use tabular number format for tables.
\newfontfamily{\tablenumbers}[Numbers={Monospaced}]{Linux Libertine O}
\newfontfamily{\libertinedisplay}{Linux Libertine Display O}

\usepackage{longtable}

\usepackage{booktabs}
\usepackage{multirow}
\usepackage{multicol}

\usepackage[justification=raggedright, labelsep=period]{caption}
\captionsetup{singlelinecheck=off}
\captionsetup{skip=0.2em}

%\usepackage{tabularx}
%\usepackage{siunitx}
\usepackage{array}
\newcolumntype{L}[1]{>{\raggedright\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{R}[1]{>{\raggedleft\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}

\newcolumntype{M}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}


\usepackage{enumitem}
\setlist{leftmargin=*}
\setlist[1]{labelindent=\parindent}
\setlist[enumerate]{label=\textsc{\alph*}., ref=\textsc{\alph*}}

\usepackage{hyperref}
%\usepackage{hanging}

\usepackage[sc]{titlesec}


\renewcommand{\solutiontitle}{\noindent}
\unframedsolutions
\SolutionEmphasis{\bfseries}

\renewcommand{\questionshook}{%
	\setlength{\leftmargin}{-\leftskip}%
}
%Change \half command from 1/2 to .5
%\renewcommand*\half{.5}


\makeatletter
\def\SetTotalwidth{\advance\linewidth by \@totalleftmargin
\@totalleftmargin=0pt}
\makeatother

\pagestyle{headandfoot}
\firstpageheader{BI 063: Evolution and Ecology}{}{\ifprintanswers\textbf{KEY}\else Name: \enspace \makebox[2.5in]{\hrulefill}\fi}
\runningheader{}{}{\footnotesize{pg. \thepage}}
\footer{}{}{}
\runningheadrule

\newcommand*\AnswerBox[2]{%
    \parbox[t][#1]{0.92\textwidth}{%
    \begin{solution}#2\end{solution}}
    \vspace{\stretch{1}}
}

\newenvironment{AnswerPage}[1]
    {\begin{minipage}[t][#1]{0.92\textwidth}%
    \begin{solution}}
    {\end{solution}\end{minipage}
    \vspace{\stretch{1}}}

\newlength{\basespace}
\setlength{\basespace}{5\baselineskip}

\newcommand{\allele}[1]{$#1$}

%\printanswers

\begin{document}

\subsection*{Evolution by genetic drift}

Evolution is be defined as genetic change in a population over time. 
The genetic change that happens in a population over time is the change 
in allele frequencies. Alleles are versions of genes that determine the 
traits of individuals, such as eye color and hair color. Frequency tells 
how common or uncommon different alleles are in a population. Common 
alleles have a high frequency while uncommon alleles have a low frequency.  

Biologists have identified five evolutionary processes that cause 
populations to evolve: genetic drift, natural selection, gene flow, 
mutation, and non-random mating. You will work in pairs to learn how 
genetic drift changes allele frequencies in populations of monstrous snarks
(\textit{Grumkin martini}). 

\bigskip

\textsc{Procedure}

\medskip

\begin{enumerate}
	\item One pair of students at your table will use a population size of 
	50 diploid individuals. The other pair will use a population size of 20
	diploid individuals. 
	
	\item Within each pair, one student will draw beans from a bag. The other
	student will record the results. You can switch tasks after every 
	generation or two, just for variety.
	
\end{enumerate}

On your table, each pair of students has two containers with light and dark beads and some 
paper sacks. The light and dark beads represent different alleles. For this 
simulation, you will use a starting frequency of 0.5 dark alleles ($D_1$, the dark 
bead) and 0.5 light alleles ($D_2$, the light bead). You will need enough beads 
to represent a population size of 20 or 50 individuals.

\begin{enumerate}[resume]
	
	\item Calculate the total number of alleles for your population.\\
	Write that number in the blank on the right side of the page. 
	\hfill \rule{0.5in}{0.4pt}\\ \emph{Remember that your
		organisms are diploid.} 
	
	\item Calculate the number of dark alleles you need for a frequency of 0.5. 
	\hfill \rule{0.5in}{0.4pt} \\ Count that number of dark beads and place 
	them in a paper sack. 
	
	\item Calculate the number of light alleles you need for a frequency of 0.5.
	\hfill \rule{0.5in}{0.4pt} \\ Count that number of light beads and add 
	them to the sack with the dark beads. Shake the sack well to mix the alleles.
	
	\item Be sure that the number of dark and light alleles adds up to the 
	total number of alleles.
	
	\item This is Generation 0. Using 0.5 each starting allele frequency, 
	calculate the genotype frequencies for Generation 0 and record them in 
	Table~\ref{tab:selection_results} on page~\pageref{tab:selection_results}. 
	\emph{Ask your instructor to check your calculations so that you start with 
		the correct number of alleles.}
	
	\item \label{sample_start} Reach into the bag and draw two alleles 
	at random. Record the genotype (\allele{DD,} \allele{Dd,} or \allele{dd}) on 
	a separate sheet of paper. Return the two alleles back to the population. 
	(Put them back in the bag with the other beads). Shake the bag well.
	
	\textsc{Note:} Returning the beads to the bag is called \textbf{sampling with replacement.} 
	Returning the alleles to the population simulates a larger population. Drawing 
	the alleles at random simulates random mating.
	
	\item Repeat Step~\ref{sample_start} until you have sampled the genotypes 
	for either 20 or 60 individuals. Record the genotype for each individual you sample. 
	Remember to return the alleles to the population after each sample. 
	
	\item Calculate the new allele and genotype frequencies, and count the number
	of individuals of each genotype. Record the results in Table~\ref{tab:selection_results} (page~\pageref{tab:selection_results}) 
	for Generation 1. 
	
\end{enumerate}

\begin{questions}
	
	\question
	Did you get \emph{exactly} the same allele frequencies for
	Generation 1 that you started with for Generation~0? What about for Generation~5? Tell how the allele frequencies changed. 
	
	\vspace*{6\baselineskip}
	
\subsubsection*{Reset the population to the new allele frequencies}

Because you used sampling with replacement, your pool of alleles that you sampled from still has a frequency of 0.5 for each 
allele. Yet, your Generation 1 population almost certainly has 
different allele frequencies.  You must therefore adjust the
allele frequencies in your paper bags to reflect the new allele 
frequencies before sampling for the next generation.

\begin{enumerate} [resume]

	\item Multiply the total number of alleles in your population by the frequency of each allele. Round to the nearest whole number. See the example below.
	
	\item Remove the existing alleles from your paper bag.
	
	\item \label{sample_stop} Add the proper number of each allele from the bag.
	
	\item Repeat steps~\ref{sample_start}â€“\ref{sample_stop} for four more generations (a total of five generations). Record the allele and genotype frequencies for each generation in Table~\ref{tab:selection_results}. %Calculate the genotype frequencies for the final generation. 
	
\end{enumerate} 

	\medskip
	
	\textsc{Example} 
	
	Assume that after sampling for Generation 1 that the frequencies of \allele{D_1} was 0.48 and \allele{D_2} was 0.52, and that your population size was 20 individuals.
	
	Number of \allele{D_1}: $0.48 \times 40 = 19.2.$ Round down to 19 alleles.
	
	Number of \allele{D_2}: $0.52 \times 40 = 20.8.$ Round up to 21 alleles.
	
	Put 19 dark alleles and 21 light alleles in the paper bag.


\question
Sketch a graph to show how the frequencies of \allele{D_1} and \allele{D_2} changed over time.

\vspace*{18\baselineskip}


Enter the frequencies for \emph{just} \allele{D_1} into a spreadsheet for all six generations from Table~\ref{tab:selection_results}. Enter the numbers all in one row. Upload the spreadsheet to the Moodle drop box indicated by your instructor. Your instructor will graph the \allele{D_1} allele frequency for all groups.

\subsubsection*{Deviation from equilibrium}

Chances are that the allele and genotype frequencies changed from generation to generation. The change might be large or small but genetic drift causes allele and genotype frequencies to change to some extent almost every generation.

The next step is to test whether Generation~5 is in Hardy-Weinberg equilibrium. You calculated the observed allele and genotype frequencies, and you counted the number of individuals of each genotype. These are \emph{observed} values because you counted them directly from your sampled data.  If Generation~5 is in Hardy-Weinberg equilibrium, then you can use your allele frequencies to predict the number of individuals of each genotype you would expect to sample. \bigskip

\textsc{Example}

\medskip

Assume you have the following data for Generation~5 for a sample of 20 individuals.

$D_1D_1$: 7 individuals  \\
$D_1D_2$: 10 individuals, and\\
$D_2D_2$: 3 individuals.

The allele frequencies are $D_1 = 0.6$ and $D_2 = 0.4$. (By now, you should know how to calculate these values on your own.) If you let $p$ equal the $D_1$ frequency, then $p^2 = 0.36$, $q^2 = 0.16$, and $2pq = 0.48$.

Next, calculate the number of \emph{expected} number of individuals of each genotype by multiplying your sample size by each genotype frequency.

$D_1D_1$: $20 \times 0.36 = 7.2$ individuals,\\
$D_1D_2$: $20 \times 0.48 = 9.6$ individuals, and\\
$D_2D_2$: $20 \times 0.16 = 3.2$ individuals.

It is \textsc{ok} to have fractions of individuals for expected numbers! \emph{Do not round the predicted numbers.} The small table below shows the comparison between your observed and predicted number of individuals.

\begin{longtable}{lrr}
	\toprule
	Genotype	& Observed	& Expected \tabularnewline
	\midrule
	$D_1D_1$ & 7  & 7.2 \tabularnewline
	$D_1D_2$ & 10 & 9.6 \tabularnewline
	$D_2D_2$ & 3  & 3.2 \tabularnewline
	\bottomrule
\end{longtable}	

The observed and expected numbers are not equal, so the population appears not to be in Hardy-Weinberg equilibrium. However, Random sampling of a small number of individuals from a large population introduces \emph{sampling error.} Sampling error does not mean that you made a mistake; instead, sampling error refers to the difference between the true value of a population (e.g., the true number of each genotype) and the value estimated from your random sample (the estimated number of each genotype). 

\subsubsection*{The $\chi^2$ test}

Could the difference between oberved and expected values be due to sampling error or could it be due to a real evolutionary process, such as genetic drift or natural selection? You can answer these questions with a statistical test called the $\chi^2$ test (chi-square; chi sounds like kye). The $\chi^2$ test compares observed values (your sample) against expected values to determine whether the differences are more likely due to random sampling error or to non-random factors, such as a violation of at least one assumption of the Hardy-Weinberg equations.
 
The equation to calculate $\chi^2$ is 

\[\chi^2 = \sum_{i=1}^{i=j}\frac{(O_i - E_i)^2}{E_i}, \]

where $O$ is the observed value, $E$ is the expected value, $i$ is the current genotype (i.e., $D_1D_1$, $ D_1D_2$, or $D_2D_2$), and $j$ is the total number of genotypes. 

The calculation is easier than it looks! For each genotype, subtract the expected value from the observe value, square that value, then divide it by the expected value. Once you have the final value for each of the three genotypes, add them together to obtain $\chi^2$. The steps are laid out in the table below. 

\question[Checkout]
Fill in each blank to calculate $\chi^2$ for Generation~5.

\begin{longtable}{cccccc}
	\toprule
	Genotype & Observed & Expected & $O-E$ & $(O-E)^2$ & $\frac{(O-E)^2}{E}$ \tabularnewline
	\midrule
	&&&&&\tabularnewline
	$D_1D_1$ & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt}\tabularnewline[2em]
	%
	$D_1D_2$ & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt}\tabularnewline[2em]
	%
	$D_2D_2$ & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt} & \rule{0.5in}{0.4pt}\tabularnewline[2em]
	&&&& $\chi^2 = $& \rule{0.5in}{0.4pt}\tabularnewline
	\bottomrule
\end{longtable}

Compare your calculated $\chi^2$ to the threshold value of $\chi^2 = 3.841$\footnote{The reasons why you use this value is beyond the scope of this exercise.}. If your calculated $\chi^2 > 3.841$, then the observed deviations from expected values is \emph{statistically significant}. A significant result means that the observed deviations are probably due to a process like genetic drift. If your calculated $\chi^2 \le 3.841$, then your results are not significant. Your deviations from expected values are probably due to random sampling error. If your results are not significant, that means your population is in Hardy-Weinberg equilibrium.

\question[Checkout]
Tell whether your results show a significant deviation from Hardy-Weinberg equilibrium and how you know. 


\end{questions}


\newpage

\begin{landscape}
%\begin{table}[t!]
\begin{longtable}[l]{C{0.75in}C{0.75in}C{0.75in}C{0.75in}C{0.75in}C{0.75in}C{0.75in}C{0.75in}C{0.75in}}
	\caption{Allele and genotype frequencies after five generations of sampling.}
	\label{tab:selection_results}\tabularnewline
	\toprule
	&
	\multicolumn{2}{c}{Allele Frequencies}	&
	\multicolumn{3}{c}{Genotype Frequencies}  &
	\multicolumn{3}{c}{Number of Individuals}\tabularnewline
	%
	\cmidrule(lr){2-3} 
	\cmidrule(l){4-6}
	\cmidrule(l){7-9}	
	%
	Generation			&
	\allele{D_1}		&
	\allele {D_2} 		&
	\allele{D_1D_1} 	&
	\allele {D_1D_2} 	&
	\allele {D_2D_2}	&
	\allele{D_1D_1} 	&
	\allele {D_1D_2} 	&
	\allele {D_2D_2}	\tabularnewline
	%
	\midrule
	& & & & & & & &\tabularnewline
	%
	0		&
	0.5	&
	0.5	&
	& %\rule{0.5in}{0.4pt}	&
	& %\rule{0.5in}{0.4pt}	&
	& %\rule{0.5in}{0.4pt}	&
	& %\rule{0.5in}{0.4pt}	&
	&  \tabularnewline[2em]%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	\tabularnewline[2em]
	%
	1	&
	\rule{0.5in}{0.4pt} &
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt} \tabularnewline[2em]
	%
	2	&
	\rule{0.5in}{0.4pt} &
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt} \tabularnewline[2em]
	%
	3	&
	\rule{0.5in}{0.4pt} &
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt} \tabularnewline[2em]
	%
	4	&
	\rule{0.5in}{0.4pt} &
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt} \tabularnewline[2em]
	%
	5	&
	\rule{0.5in}{0.4pt} &
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt}	&
	\rule{0.5in}{0.4pt} \tabularnewline
	%
	%6	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	\tabularnewline
	%7	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	\tabularnewline[2em]
	%8	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	&
	%\rule{0.5in}{0.4pt}	\tabularnewline[2em]
	\bottomrule 
\end{longtable}
%\end{table}

\end{landscape}

\end{document}  