%!TEX TS-program = lualatex
%!TEX encoding = UTF-8 Unicode

\documentclass[12pt, hidelinks]{exam}

%\printanswers


\usepackage{graphicx}
	\graphicspath{{/Users/goby/Pictures/teach/163/lab/}
		{/Users/goby/Pictures/teach/300/exercises/}
	{img/}} % set of paths to search for images

\usepackage{geometry}
\geometry{letterpaper, left=1.5in, bottom=1in}                   
%\geometry{landscape}                % Activate for for rotated page geometry
\usepackage[parfill]{parskip}    % Activate to begin paragraphs with an empty line rather than an indent
\usepackage{amssymb, amsmath}
\usepackage{mathtools}
	\everymath{\displaystyle}

\usepackage{fontspec}
\setmainfont[Ligatures={TeX}, BoldFont={* Bold}, ItalicFont={* Italic}, BoldItalicFont={* BoldItalic}, Numbers={OldStyle}]{Linux Libertine O}
\setsansfont[Scale=MatchLowercase,Ligatures=TeX, Numbers=OldStyle]{Linux Biolinum O}
\setmonofont[Scale=0.82]{Linux Libertine Mono O}
\usepackage{microtype}

\newfontfamily{\tablenumbers}[Numbers={Monospaced}]{Linux Libertine O}

\usepackage{pdflscape}

% To define fonts for particular uses within a document. For example, 
% This sets the Libertine font to use tabular number format for tables.
 %\newfontfamily{\tablenumbers}[Numbers={Monospaced}]{Linux Libertine O}
% \newfontfamily{\libertinedisplay}{Linux Libertine Display O}

\usepackage{booktabs}

\usepackage{caption}
\captionsetup{labelsep=period} % Removes colon following figure / table number.

\usepackage{multicol}

\usepackage{adjustbox}

\usepackage{tikz}
\usetikzlibrary{backgrounds}

\usepackage{forest}
\forestset{
  mytree/.style={
    for tree={
      edge+={ultra thick},
      edge path'={
        (!u.parent anchor) -| (.child anchor)
      },
      grow'=north,
      parent anchor=children,
      child anchor=parent,
      anchor=base,
      l sep=1cm,
      s sep=2mm,
      if n children=0{tier=word, align=center, base=bottom}{coordinate, tree node}
    }
  }
}
\forestset{
  declare dimen register={timeline offset},
  timeline offset'=10mm,
  declare toks register={timeline target},
  timeline target=,
  declare boolean={tree node}{0},
  timelinetree/.style={
    for tree={
      edge+={ultra thick},
      edge path'={
        (!u.parent anchor) -| (.child anchor)
      },
      grow=north,
      parent anchor=children,
      child anchor=parent,
      anchor=base,
%      l sep=1cm,
      s sep=2mm,
      if n children=0{tier=word, align=center, base=bottom, not tree node}{coordinate, tree node}
    }
  },
%
	age/.style={
		tikz+={
			\begin{scope}[on background layer] 
				\draw [gray, thick, dotted] () -- ( -| timeline base) node[black,anchor=east]{#1}; 
			\end{scope} 
		}
	},
%
  timeline/.style={
    before drawing tree={
      timeline target/.option=name,
      tempdima/.option=y,
      for tree={
        if={>OR>{y}{tempdima}}{timeline target/.option=name}{},
      }
    },
    tikz+={
      \begin{scope}[on background layer]
        \draw ([xshift=-\foresteregister{timeline offset}]current bounding box.west |- .parent anchor) coordinate (timeline base) -- (\foresteregister{timeline target}.child anchor -| timeline base) node [above] {\textsc{mya}};
      \end{scope}
    },
  },
}

\usepackage{longtable}
%\usepackage{siunitx}
\usepackage{array}

\newcolumntype{L}[1]{>{\raggedright\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{R}[1]{>{\raggedleft\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}

\usepackage{enumitem}
%\setenumerate{label=\Alph.}
\setlist{leftmargin=*}
\setlist[1]{labelindent=\parindent}
\setlist[enumerate]{label=\textsc{\alph*}.}
\setlist[itemize]{label=\color{gray}\textbullet}

%\usepackage{caption}
%	\captionsetup{labelsep=period, justification=raggedright} % Removes colon following figure / table number.
%	\captionsetup{singlelinecheck=off}
%	\captionsetup[table]{skip=0pt}

\usepackage{hyperref}
%\usepackage{placeins} %PRovides \FloatBarrier to flush all floats before a certain point.
\usepackage{hanging}

\usepackage[sc]{titlesec}

%% Commands for Exam class
\renewcommand{\solutiontitle}{\noindent}
\unframedsolutions
\SolutionEmphasis{\bfseries}

\renewcommand{\questionshook}{%
	\setlength{\leftmargin}{-\leftskip}%
}

\newcommand{\hidepoints}{%
	\pointsinmargin\pointformat{}
}

\newcommand{\showpoints}{%
	\nopointsinmargin\pointformat{(\thepoints)}
}

%Change \half command from 1/2 to .5
\renewcommand*\half{.5}

\pagestyle{headandfoot}
\firstpageheader{\textsc{bi}\,063 Evolution and Ecology}{}{\ifprintanswers\textbf{KEY}\else Name: \enspace \makebox[2.5in]{\hrulefill}\fi}
\runningheader{}{}{\footnotesize{pg. \thepage}}
\footer{}{}{}
\runningheadrule

\newcommand*\AnswerBox[2]{%
    \parbox[t][#1]{0.92\textwidth}{%
    \begin{solution}#2\end{solution}}
%    \vspace*{\stretch{1}}
}

\newenvironment{AnswerPage}[1]
    {\begin{minipage}[t][#1]{0.92\textwidth}%
    \begin{solution}}
    {\end{solution}\end{minipage}
    \vspace*{\stretch{1}}}

\newlength{\basespace}
\setlength{\basespace}{5\baselineskip}

\newcommand{\dna}{\textsc{dna}}

\begin{document}

\subsection*{Use dna sequence to make phylogenetic trees}

You have used homologous characters to construct phylogenetic trees that 
show the relationships among various taxonomic groups. Another method
that can be used to build phylogenetic trees is by comparing \dna{} sequences.
D\textsc{na} is found in all living organisms, from vertebrates to bacteria. As 
organisms evolve, mutations cause their \dna{} to change. The mutations
accumulate in the related lineages since they last shared a common ancestor.
Species that shared a common ancestor recently will have relatively few mutational
differences. Species that shared a common ancestor farther back in time will
have more mutational differences. Thus, the number of mutational differences
can be used to construct phylogenetic trees.

%Earlier, you compared the \dna{} sequence from seven species of made-up birds. 
%The \dna{} sequence from each species was only 20 nucleotides long. That was useful
%to show you how to use \dna{} sequences to make a phylogenetic tree. But, real studies
%often use \dna{} sequences that are several hundred to several thousand nucleotides long
%from dozens of species. Trying to find all of those differences by eye would be nearly impossible
%to do accurately. On the other hand, hundreds of sequences can be converted into a phylogenetic tree very 
%rapidly by computer. 

For this exercise, you will use a common analytical tool to build two phylogenetic trees using \dna{} sequence
from two different genes. Each tree will contain representative species for the major groups of organisms you have
used in the last few labs. After you have the trees, you will draw a complete phylogeny and then map on the
various homologies you have studied to see how the \dna{} sequence data and the homological data support one another.

You will analyze two genes, one at a time, using your laptop computer or one that has been provided for you (log in with your \textsc{se k}ey).   Work in pairs (or three if necessary) to complete the following \ref{final_step} steps for \emph{each} gene.

\begin{enumerate}

	\item Open a web browser. Go to \url{http://mtaylor4.semo.edu/~goby/bi163/}. Click on one of the gene names (18\textsc{s} or \textsc{coi}).\footnote{18\textsc{s} is a gene that encodes the small subunit of ribosomes used to translate m\textsc{rna} to make proteins. C\textsc{oi} is part of a gene that encodes an enzyme used in aerobic respiration.}% \textsc{rag}1 is part of a gene that encodes a protein used in the vertebrate immune system.} 
	
	\item Copy the \emph{entire} sequence set (select all, or ctrl/cmd-A). Remember which sequence you copied because you will need to know in a few more steps.
	
%	\item Go back to the list of genes. Click on the \textsc{ra}x\textsc{ml} link. This will take you to the \textsc{ra}x\textsc{ml} site\footnote{\textsc{ra}x\textsc{ml} uses a technique called maximum likelihood, a mathematically intensive method that finds the best phylogenetic tree based on different models of \dna{} evolution.} to analyze the \dna{} sequences. 

	\item Go back to the list of genes. Click on the “Phylogenetic Analysis Website” link to visit the site where you will analyze the \dna{} sequences \ref{fig:home_page}).\footnote{This site uses a technique called maximum likelihood, a mathematically intensive method that finds the best phylogenetic tree based on different models of \dna{} evolution.} 

  	\item Scroll down just a bit until you see the link called “Advanced,” then click it.

	\item Create your workflow (Fig.~\ref{fig:workflow}). The workflow consists of several steps
that you need to obtain your final tree. You do not need your alignment curated so uncheck Gblocks and
then click the “Create Workflow” button.

	\item Paste the \textsc{dna} sequences that you copied earlier into the area
marked “Input Data” (Fig.~\ref{fig:paste_sequences}) and then scroll down to set the
parameters for the PhyML phylogenetic analysis (Fig.~\ref{fig:phyml}).

\begin{center}
	\includegraphics[width=0.7\textwidth]{fig3_home_page}
	\captionof{figure}{Home page of Phylogeny.fr, an online site for performing phylogenetic analyses.\label{fig:home_page}}
\end{center}

\vfill

\begin{center}
	\includegraphics[width=0.6\textwidth]{fig4_workflow}
	\captionof{figure}{Workflow settings.\label{fig:workflow}}
\end{center}


\newpage

\begin{center}
	\includegraphics[width=0.7\textwidth]{fig5_paste_sequences}
	\captionof{figure}{Input data for your \textsc{dna} sequences.\label{fig:paste_sequences}}
\end{center}

\bigskip

\bigskip

\begin{center}
	\includegraphics[width=0.9\textwidth]{fig6_phyML_settings}
	\captionof{figure}{PhyML settings for the phylogenetic analysis.\label{fig:phyml}}
\end{center}

	\item  Set the substitution model to “HKY{\tablenumbers 85} (\textsc{dna/rna})” and then click the “Submit” button near the bottom of the page.

	\item After a few minutes, a phylogenetic tree will appear. Scroll down to section labeled “Select an action and click leaf or internal branch,” and then click the “Reroot (outgroup)” button (Fig.~\ref{fig:reroot}).  Then, scroll back up to the tree to click on the taxon name in the tree, according to this table:
	
	{\centering\begin{tabular}{@{}ll@{}}
		\toprule
		Gene &	Outgroup name\\
		\midrule
		18\textsc{s}	&  bacteria\\
		\textsc{coi}	& mollusk\\
		%		\textsc{rag}1	& bass\\
		\bottomrule		
	\end{tabular}\par}
	
	This will reroot the phylogeny with the chosen group in the basal position.

\begin{center}
	\includegraphics[width=0.7\textwidth]{fig_reroot}
	\captionof{figure}{Reroot to chosen outgroup.\label{fig:reroot}}
\end{center}
	
	\item 
	\textbf{Show this tree to your instructor.} If successful, your instructor will give you a copy of the tree to work with for the final part of this exercise. \label{final_step}

	\item Click on the “Home” button (near upper left of page) to repeat this process for the other gene.


\end{enumerate}

	
	
	

\begin{questions}

\question
Now that you have copies of both phylogenetic trees, assemble them into one large tree with all taxonomic groups. The tree from each gene has representatives for \emph{some} but not all of the taxonomic groups. Each tree “overlaps” partly with the other tree. With careful inspection, you should be able to see how the two trees piece together. 

\textsc{Note:} Many taxonomic groups have more than one representative (e.g., bird1, bird2, etc.). You only need to draw one branch for the group (e.g., birds).

\question[Checkout]
The table on the next page contains a list of homologies for the taxonomic groups, many of them studied in previous exercises. For each homology, make a tick mark at the appropriate common ancestor on your final phylogeny where that homology first evolved. Write the letter code next to the tick mark. \textit{Ignore Paramecium.} Once done, ask your instructor to check your work.

\end{questions}

\newpage

\thispagestyle{empty}

\begin{landscape}

\begin{longtable}{@{}L{1in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}L{0.25in}@{}}
\toprule
Taxon		& A	& B	& C	& D	& E	& F	& G	& H	& I	& J	& K	& L	& M & N	\tabularnewline
\midrule
Bacteria	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0 & 1	\tabularnewline
Paramecium	& 1	& 0	& 0	& 1	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0 & 1	\tabularnewline
Plants		& 1	& 1	& 1	& 1	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0 & 1	\tabularnewline
Algae		& 1	& 1	& 1	& 1	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0 & 1	\tabularnewline
Fungi		& 1	& 0	& 0	& 1	& 1	& 0	& 0	& 0	& 0	& 0	& 0	& 0	& 0 & 1	\tabularnewline
Mollusks	& 1	& 0	& 0	& 1	& 1	& 1	& 1	& 1	& 0	& 0	& 0	& 0	& 0 & 1	\tabularnewline
Annelids	& 1	& 0	& 0	& 1	& 1	& 1	& 1	& 1	& 0	& 0	& 0	& 0	& 0 & 1	\tabularnewline
Arthropods	& 1	& 0	& 0	& 1	& 1	& 1	& 0	& 1	& 0	& 0	& 0	& 0	& 0 & 1	\tabularnewline
Echinoderms	& 1	& 0	& 0	& 1	& 1	& 1	& 0	& 0	& 1	& 0	& 0	& 0	& 0 & 1	\tabularnewline
Mammals		& 1	& 0	& 0	& 1	& 1	& 1	& 0	& 0	& 1	& 1	& 1	& 1	& 0 & 1	\tabularnewline
Reptiles	& 1	& 0	& 0	& 1	& 1	& 1	& 0	& 0	& 1	& 1	& 1	& 1	& 1 & 1	\tabularnewline
Birds		& 1	& 0	& 0	& 1	& 1	& 1	& 0	& 0	& 1	& 1	& 1	& 1	& 1 & 1	\tabularnewline
Amphibians	& 1	& 0	& 0	& 1	& 1	& 1	& 0	& 0	& 1	& 1	& 1	& 0	& 0 & 1	\tabularnewline
Fishes		& 1	& 0	& 0	& 1	& 1	& 1	& 0	& 0	& 1	& 1	& 0	& 0	& 0 & 1	\tabularnewline

\bottomrule
\end{longtable}\label{presence_table}

Key to letter codes:

\begin{multicols}{3}
	\raggedcolumns
	\begin{enumerate}
		\item Cell with nucleus.
		\item Cells with chloroplasts.
		\item Cell walls composed of cellulose.
		\item Cells with mitochondria.
		\item Cellular production of chitin.
		\item Cell walls absent.
		\item Trochophore larvae.
		\item Protostome development.
		\item Deuterostome development.
		\item Internal skeleton with vertebrae.
		\item Tetrapods.
		\item Amniotic egg.
		\item Diapsid skulls.
		\item DNA for genetic information.
	\end{enumerate}
\end{multicols}


\newpage

\thispagestyle{empty}


\ifprintanswers
\begin{center}

\adjustbox{width=\linewidth}{
\begin{forest}
mytree,
[
 [,name=root
  [bacteria]
  [,name=eukaryotes
   [\textit{Paramecium}]
    [
     [
      [,name=chloro
       [algae]
       [plants]
      ]
     ]
     [,name=unikonts
      [fungi]
      [,name=animals
       [,name=protostomes
        [,name=trochophore
         [mollusks]
         [annelids]
        ]
	    [
		 [arthropods]
	    ]
	   ]
	   [,name=deuterostomes
	    [echinoderms]
   	    [,name=vertebrates
   	     [fishes]
         [,name=tetrapods
          [
           [amphibians]
          ]
          [,name=amniotes
           [mammals]
           [,name=reptiles
            [birds]
            [reptiles]
           ]
          ]
         ]
        ]
       ]
      ]
     ]
    ]
   ]
  ]
 ]
\filldraw (root) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{n}}};
\filldraw (eukaryotes) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{a,d}}};
\filldraw (chloro) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{b, c}}};
\filldraw (unikonts) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{e}}};
\filldraw (animals) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{f}}};
\filldraw (protostomes) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{h}}};
\filldraw (trochophore) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{g}}};
\filldraw (deuterostomes) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{i}}};
\filldraw (vertebrates) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{j}}};
\filldraw (tetrapods) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{k}}};
\filldraw (amniotes) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{l}}};
\filldraw (reptiles) circle [radius=3pt, fill=black, yshift=-5mm] node [left, xshift=-1mm, yshift=-5mm] {{\small \textsc{m}}};
\end{forest}}
\end{center}
\fi

\end{landscape}

\end{document}  